test/server/lib/DocApi.ts

```diff
@@ -4100,6 +4105,26 @@ function testDocApi() {
         );
 
       });
+
+      it("doesn't send trigger webhook that has been disabled", async function () {
+        // Create a test document.
+        const ws1 = (await userApi.getOrgWorkspaces('current'))[0].id;
+        const docId = await userApi.newDoc({name: 'testdoc'}, ws1);
+        const doc = userApi.getDocAPI(docId);
+
+        await createWebhooks({
+          docId, tableId: 'Table1', isReadyColumn: "B", eventTypesSet: [ ["add"] ], enabled: false
```

Comment:
Please add a similar but enabled webhook and confirm that there is an event for the enabled webhook. I'm worried that this test could pass for the wrong reasons, e.g. because of a race condition.

Reply:
I have made 2 different tests with the same codebase. Is it fine or you would like this in a same single test run?